rotkehlchen/assets/asset.py

```diff
@@ -916,6 +918,23 @@ def to_dict(self) -> Dict[str, Any]:
                 'underlying_tokens': underlying,
             }
 
+        if self.is_custom_asset():
+            custom_asset = CustomAsset.from_identifier(self.identifier)
```

Comment:
I see that this approach is also used for evm token serialization but why do we have to load the custom asset from the db here? Why can't we just convert it to `CustomAsset` type?